src/main/webapp/3Dscript.js

```diff
@@ -194,8 +199,39 @@ function gui() {
   gui.add(poseTransform, 'scale', .5, 2, .25)
       .onChange(plotOrientation)
       .onFinishChange(plotTrajectory).name('Pose Scale Multiplier');
+  /* Time value. */
+  const timeStart = {time: 'no point yet'};
+  time = gui.add(timeStart, 'time');
```

Comment:
I know this was pre-existing, but it's confusing to have both a variable and a function named `gui`.